### ISSUE TITLE
Connect Job Detail to Backend

### ISSUE THREAD

**Issue by jaredwebber** [Speuce/JAMCo#53]
Following #32 - Connect Frontend to Backend once constructed

**Comment by Speuce** [comment]
Blocked by #30

**Comment by jaredwebber** [comment]
Ticket may be redundant with #45  - likely easiest to handle all connections in a single ticket as most (if not all) of the interaction will be from `JobTrackingView.vue`

**Comment by jaredwebber** [comment]
Merged into #45
